src/app.js

```diff
@@ -306,6 +319,23 @@ TunnelService.switchToHttps = function() {
   startHttpsGateway();
 };
 
+// This part starts our Service Discovery process.
+// We check to see if mDNS should be setup in default mode, or has a previous
+// user setup a unique domain. Then we start it.
+mDNSserver.getmDNSstate().then((state) => {
+  try {
+    mDNSserver.getDNSconfig().then((mDNSconfig) => {
+      console.log(`DNS config is: ${mDNSconfig.host}`);
+      mDNSserver.server.changeProfile(mDNSconfig);
+      mDNSserver.server.startService(state);
+    });
+  } catch (err) {
+    // if we failed to startup mDNS it's not the end of the world log it
+    // and carry on
+    console.error('Service Discover failed to start with error: ${err}');
```

Comment:
You should be using backticks here instead of single quotes.

Reply:
thank you. done.